### PR TITLE
Dark search popover

### LINKED ISSUE #737: Search results popover: should respect dark mode
https://docsearch.algolia.com/docs/styling/

Bildschirmfoto 2020-10-14 um 13 15 18

### PR DESCRIPTION
**Pull request by mischnic**
Closes https://github.com/parcel-bundler/website/issues/737

Bildschirmfoto 2020-11-14 um 21 58 32

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -14,7 +14,12 @@ module.exports = function (eleventyConfig) {
   eleventyConfig.addPlugin(eleventyNavigationPlugin);
   eleventyConfig.setDataDeepMerge(true);
 
-  eleventyConfig.addPlugin(pluginSass, { watch: ["src/*.{scss,sass}"] });
+  eleventyConfig.addPlugin(pluginSass, {
+    watch: ["src/*.{scss,sass}"],
+    cleanCSSOptions: {
```
Comment (mischnic) [inline .eleventy.js]: that css minified would also remove the first newly added rule 🤷

File: src/doc-styles.scss [hunk 2]
```diff
@@ -804,3 +804,47 @@ body {
     }
   }
 }
+
+html.dark {
+  .algolia-autocomplete {
```
Comment (DeMoorJasper) [inline src/doc-styles.scss]: Can't you add `/* clean-css ignore:start */` above this line and `/* clean-css ignore:end */` at the end of this css block to skip removal of these lines?

Not sure if we should completely opt-out from optimising for this little bit of css

**Review by DeMoorJasper — CHANGES_REQUESTED** [review]

File: src/doc-styles.scss (hunk 2 above)
Comment (mischnic) [reply src/doc-styles.scss]: > Not sure if we should completely opt-out from optimising for this little bit of css

I've reordered the rules and cleancss seems to work correctly now.

The difference was surprisingly small:
disabled: 19kb
level 0: 16kb
level 1 (default): 15kb

Comment (DeMoorJasper) [reply src/doc-styles.scss]: oh awesome, probably means we don't have a lot of unnecessary css?

**Review by DeMoorJasper — APPROVED** [review]

**Pull request merged**